protocols/luci-proto-wireguard/htdocs/luci-static/resources/protocol/wireguard.js

```diff
@@ -193,6 +193,8 @@ return network.registerProtocol('wireguard', {
 			return true;
 		};
 
+		o = s.taboption('advanced', form.DynamicList, 'ip6prefix', _('IPv6 routed prefix'), _('This is the prefix routed to you by the tunnel broker for use by clients'));
```

Comment:
Change to:
`This is the prefix routed to you by your provider for use by clients`